app/src/gui/components/notebook/index.tsx

```diff
@@ -233,100 +276,120 @@ export default function NotebookComponent(props: NotebookComponentProps) {
+                  color="primary"
+                  sx={{marginLeft: '4px'}}
                 >
-                  <Typography variant={'h6'} sx={{m: 2}} gutterBottom>
-                    About
-                  </Typography>
-                  <Table size={'small'}>
-                    <TableBody>
-                      <TableRow>
-                        <TableCell>
-                          <Typography variant={'overline'}>Status</Typography>
-                        </TableCell>
-                        <TableCell>
-                          <MetadataRenderer
-                            project_id={project.project_id}
-                            metadata_key={'project_status'}
-                            chips={false}
-                          />
-                        </TableCell>
-                      </TableRow>
-                      <TableRow>
-                        <TableCell>
-                          <Typography variant={'overline'}>
-                            Lead Institution
-                          </Typography>
-                        </TableCell>
-                        <TableCell>
-                          <MetadataRenderer
-                            project_id={project.project_id}
-                            metadata_key={'lead_institution'}
-                            chips={false}
-                          />
-                        </TableCell>
-                      </TableRow>
-                      <TableRow>
-                        <TableCell>
-                          <Typography variant={'overline'}>
-                            Project Lead
-                          </Typography>
-                        </TableCell>
-                        <TableCell>
-                          <MetadataRenderer
-                            project_id={project.project_id}
-                            metadata_key={'project_lead'}
-                            chips={false}
-                          />
-                        </TableCell>
-                      </TableRow>
-                      <TableRow>
-                        <TableCell>
-                          <Typography variant={'overline'}>
-                            Last Updated
-                          </Typography>
-                        </TableCell>
-                        <TableCell>
-                          <MetadataRenderer
-                            project_id={project.project_id}
-                            metadata_key={'last_updated'}
-                            chips={false}
-                          />
-                        </TableCell>
-                      </TableRow>
-                    </TableBody>
-                  </Table>
-                </TableContainer>
-              </Grid>
-              <Grid item xs={12} sm={12} md={12} lg={4}>
-                <RangeHeader
-                  project={project}
-                  handleAIEdit={handleNotebookTabChange}
+                  Edit
```

Comment:
Hide the edit button until we can actually edit.

Reply:
done